pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1362,6 +1362,17 @@ func parseOneSystemAdapterConfig(getconfigCtx *getconfigContext,
 		log.Errorf("parseSystemAdapterConfig: %s", errStr)
 		port.RecordFailure(errStr)
 	}
+	// Make sure that even without wireless network config attached,
+	// EVE microservices properly recognize wireless network ports
+	// using the WType attribute.
+	if port.WirelessCfg.IsEmpty() && phyio != nil {
+		switch phyio.Ptype {
```

Comment:
Is it ok that we ignore the other cases here?
I see that `phyio` may be of the other types as well... We just checked it's a net type here https://github.com/lf-edge/eve/blob/94f58cd41b7a429cf0723820b1d859a7bd2c38a0/pkg/pillar/cmd/zedagent/parseconfig.go#L1214, but after that it may be one of 5 different types, and here we handle only 2. Or are the other types to be ignored as they have nothing to do with the wireless connection?